test/TestUtils.py

```diff
@@ -185,3 +187,48 @@ def test_expand_paths_vars(monkeypatch):
     monkeypatch.setenv('TEST_PATH', test_path)
     assert utils.expand_paths_vars(['~']) == [os.path.expanduser('~')]
     assert utils.expand_paths_vars(['$TEST_PATH']) == [test_path]
+
+
+def test_get_yaml_files_no_git_verbose(monkeypatch, capsys):
+    options, _ = cli.get_config(['-v'])
+    monkeypatch.setenv('PATH', '')
+    utils.get_yaml_files(options)
+    msg1, msg2 = capsys.readouterr().out.splitlines()[:2]
+    assert msg1 == 'Discovering files to lint: git ls-files *.yaml *.yml'
+    assert msg2.startswith('Warning: Failed to locate command:')
+
+
+def test_get_yaml_files_not_git_verbose(monkeypatch, capsys):
+    options, _ = cli.get_config(['-v'])
+    monkeypatch.chdir('/tmp')
+    utils.get_yaml_files(options)
+    msg1, msg2 = capsys.readouterr().out.splitlines()[:2]
+    assert msg1 == 'Discovering files to lint: git ls-files *.yaml *.yml'
+    assert msg2.startswith('Warning: Failed to discover yaml files to lint using git:')
+
+
+@pytest.mark.parametrize('is_in_git', (True, False))
```

Comment:
On the second thought, it's probably worth labeling params:
```suggestion
@pytest.mark.parametrize(
    'is_in_git',
    (True, False),
    ids=('in Git', 'outside Git'),
)
```